pkg/operator/controller/gatewayclass/subscription.go

```diff
@@ -58,7 +58,7 @@ func desiredSubscription(name types.NamespacedName) (*operatorsv1alpha1.Subscrip
 			Channel:                "stable",
 			InstallPlanApproval:    operatorsv1alpha1.ApprovalAutomatic,
 			Package:                "servicemeshoperator",
-			CatalogSource:          "redhat-operators",
+			CatalogSource:          "quay-iib",
```

Comment:
Is this just for testing?

Reply:
Yes, it's the only way we can get 2.5 images right now.